main.go

```diff
@@ -9,9 +9,12 @@ import (
 func main() {
 	mx := http.NewServeMux()
 
-	logFile, _ := os.Create("log.log")
-	core := Core{lg: slog.New(slog.NewJSONHandler(logFile, nil))}
-	api := API{core: &core}
+	coreLogFile, _ := os.Create("core_log.log")
+	apiLogFile, _ := os.Create("api_log.log")
+	core := Core{lg: slog.New(slog.NewJSONHandler(coreLogFile, nil))}
+	api := API{core: &core, lg: slog.New(slog.NewJSONHandler(apiLogFile, nil))}
```

Comment:
Обычно делают так

```suggestion
	logFile, _ := os.Create("log.log")

	lg := slog.New(slog.NewJSONHandler(logFile, nil))

	core := Core{lg: lg.With("module", "core")}
	api := API{core: &core, lg: lg.With("module", "api")}
```

Тогда у вас один файл ==> его проще обслуживать. + Есть вся хронология событий с разделением по модулям